### ISSUE TITLE
Update plugin documentation for release

### ISSUE THREAD

**Issue by soumagne** [mercury-hpc/mercury#249]

**Comment by ael-code** [comment]
This issue has been closed but no information have been provided about how has been fixed. Any commit reference?

**Comment by soumagne** [comment]
@ael-code yes please have a look at http://mercury-hpc.github.io/documentation/#available-plugins please let me know if anything is not clear
